CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ _ `_optional` subpackage for managing optional dependencies
    convenience constructors
 - `DiscreteParameter.to_subspace`, `ContinuousParameter.to_subspace` and
   `Parameter.to_searchspace` convenience constructors
+- Example for the `register_hook` utility
```

Comment:
Make it concrete

Reply:
How about this one? 
```suggestion
Example for calculating and plotting the PI using the `register_hook` utility
```

Reply:
Problems:
* The PI abbreviation should not be used in the changelog since not define
* The utility is actually called `register_hooks`
```suggestion
- Example for monitoring the probability of improvement using `register_hooks`
```